### DISCUSSION TITLE
Welcome to XUpdate Discussions!

### DISCUSSION THREAD

**Discussion by xuexiangjys** [xuexiangjys/XUpdate#117] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JungleDeeer** [comment]
你好大神，我正在学习安卓开发，有幸遇见您的XUpdate，但在使用中遇到了无法下载更新包的问题，报错写的是SSL handshake aborted: ssl=0xdd3647d8: I/O error during system call, Connection reset by peer，搜索后发现大部分是说这是对ssl版本支持的问题，我服务器端给的下载地址就是github的release发布地址，是tls1.3，能麻烦提供下问题解决建议吗，谢谢！

**Reply by xuexiangjys** [reply]
最好存储到云服务上去,github的下载地址不是很稳定

**Comment by JungleDeeer** [comment]
好的，谢谢!

薛翔 ***@***.***> 于2021年3月18日周四 上午12:56写道：
